src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-tapChkBx.html

```diff
@@ -0,0 +1,5 @@
+<div>
+<br>
+Generate a TAP style test report, named <b> taptestresults.tap, </b> to be stored in the<b> matlabTestArtifacts</b> folder of the current Jenkins workspace.
```

Comment:
Remove extraneous spaces around `taptestresults.tap,`

`named <b> taptestresults.tap, </b> to` -> `named <b>taptestresults.tap</b>, to`

Reply:
Done.